src/library.js

```diff
@@ -1427,6 +1474,8 @@ const createPilingJs = (rootElement, initOptions = {}) => {
     const stateUpdates = new Set();
     const newlyCreatedItems = [];
 
+    const changedPiles = [];
```

Comment:
```suggestion
    const updatedPiles = [];
```

Just for semantic consistency :)